Antonio_Ramos_Documentation

```diff
@@ -0,0 +1,171 @@
+
+- attempted to run program again with the console returning the following message
+"
+Traceback (most recent call last):
+
+  File "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\K-function Local_Scipy.py", line 41, in <module>
+    pointFeatures = shapefile.Reader(pFC)
+
+  File "C:\Apps-SU\Anaconda3\lib\site-packages\shapefile.py", line 553, in __init__
+    self.load(args[0])
+
+  File "C:\Apps-SU\Anaconda3\lib\site-packages\shapefile.py", line 646, in load
+    raise ShapefileException("Unable to open %s.dbf or %s.shp." % (shapeName, shapeName))
+
+ShapefileException: Unable to open G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.dbf or G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp.
+"
+
+- found file "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp" however when I tried to open it with python code I used earlier, I got the following error message
+"
+DriverError: G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp: No such file or directory
+"
+
+- noticed that within the file "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data" there are updated versions of the "GoodLabs.shp" and "GoodLabs.dbf" named "GoodLabs_New.shp" and "GoodLabs_New.dbf" will update the file names within the K-function Local_Scipy.py script to the new names.
+
+- rewrote the path name on line 39 to account for the updated GoodLabs.
+old od - "pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp""
+new code - pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs_New.shp""
+
+- ipython console displays the run file command but then doesnt return anything and seems to freeze since it doesnt let me run anyother scripts, will attempt to restart and run again
+
+- again no response within the console and unabe to run other scripts
+
+- reverted path name to original - " pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp""
+and got shapefile exception again when trying to run the script 
+
+- realized I was working on the wrong script and started working on the K-function Local_3Stage.py located inG:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions
+
+- once I started working on the correct script I realized that the Will's dosumentation is not up to date and does not reference the same files as the script does. So, I created an excell spreadsheet to list all the files mentioned by Will in his documentation in an attemtp to better understand where everything was coming from, what it did, and how it should function with the python scripts. 
+
+- emailed Will asking if he had any more updated documentation. Will responded with the most comprehensive documentation that he has, however it still does not mention the files being refrenced by the K-function Local_3Stage.py script. 
```

Comment:
Add where you put this file at least...if it's more current than what we have on Github now, then please put it in your local repository and create a pull request so I can integrate it into what we have in the centralized repo

Reply:
It is the same one we have on github but I will add a note about this.